src/main/java/mate/academy/model/dto/UserResponseDto.java

```diff
@@ -3,24 +3,15 @@
 import java.util.List;
 
 public class UserResponseDto {
-    private String email;
-    private String password;
+    private String login;
```

Comment:
in User is used email field, not logic, so return the original version or change naming in User